code/game/objects/items/holy_weapons.dm

```diff
@@ -311,12 +311,12 @@
 /obj/item/nullrod/claymore
 	icon_state = "claymore_gold"
 	inhand_icon_state = "claymore_gold"
-	worn_icon_state = "claymore"
+	worn_icon_state = "claymore_gold"
 	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
 	name = "holy claymore"
 	desc = "A weapon fit for a crusade!"
-	w_class = WEIGHT_CLASS_HUGE
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
will this interfere with storing these in your chaplain armor's suit storage slot?

Reply:
This actually allows you to place it there in the first place.